src/cc1101.cpp

```diff
@@ -622,6 +626,9 @@ void cc1101::getRxFifo(uint16_t Boffs) {           // xFSK
 					MSG_PRINT(F(";R="));
 					MSG_PRINT(RSSI);
 					MSG_PRINT(';');
+					MSG_PRINT(F(";A="));
+					MSG_PRINTtoHEX(freqErr);
```

Comment:
Warum den Wert in hex ausgeben?
Mir erscheint die Ausgabe wie beim RSSI in dezimal besser. Da sehe ich sofort, ob ich mit der Frequenz hoch oder runter muss:

`MN;D=5F79E6ABBA8A18AAAAAAAABDD8AAB89A92A8EBF9ABAAAAAAAA00;R=197;A=-6;`

Reply:
Wollte die 8 Bit halt mit zwei Zeichen und ohne Vorzeichen realisieren.

Reply:
Naja, bei FSK müssen wir uns ja über die serielle Ausgabe weniger Gedanken machen. Der Prozessor langweilt sich sowieso die meiste Zeit, weil der CC1101 die Dekodierung übernimmt. Die Übergabe in dezimal spart uns zweimal die Umrechnung.

Reply:
Mir ging es um die Umrechnung und Validierung. Das klappt halt bei Hexadezimal ganz gut und ist schon implementiert.
Ich habe es jetzt mal als unsinged 8bit Integer ausgegeben.

Reply:
Die Umrechnung kannst du dir sparen:
```
int8_t freqErr = cc1101::readReg(0x32, CC1101_READ_BURST); // 0x32 (0xF2): FREQEST – Frequency Offset Estimate from Demodulator
```
Ausgabe:
```
MSG_PRINT(F(";A="));
MSG_PRINT(freqErr);
```
ergibt:
```
MN;D=EEF46CF7FFFFFFEDEDC6FFFF7F110B9308000000121239000080;R=24;A=4;
MN;D=EDBEE6ABA2EA18AAAAAAAA8B9CAAA38A3DAAAAAAAAAAAAAAAA00;R=185;A=-3;
```
Umrechnung in kHz in der SIGNALduino.pm:
```
  if ( defined $msg_parts{freqafc} ){ # AFC cc1101 0x32 (0xF2): FREQEST – Frequency Offset Estimate from Demodulator
    $freqafc = $msg_parts{freqafc};
    $freqafc = int(26000000 / 16384 * $freqafc / 1000); # Ausgabe in kHz
  }
```

Wenn du noch etwas Geduld hast, ich bin jetzt soweit, meine Version zu veröffentlichen. Ich habe die Geschichte noch um eine AFC erweitert.

Reply:
Du kannst es ja ergänzen. Ich hab den PR ja schon gemacht.